### PR TITLE
Explicitly set vm box

### PR DESCRIPTION
**Pull request by andy-dufour**

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -13,5 +13,5 @@ platforms:
 suites:
   - name: default
     run_list:
-      - recipe[chef_server::default]
+      - recipe[chef_delivery::default]
```
Comment (ncerny) [inline .kitchen.yml]: why chef_delivery?

**Comment by stephenlauck** [comment]
:shipit:

File: recipes/default.rb [hunk 2]
```diff
@@ -24,6 +28,10 @@
 chef_server 'fe01.vagrantup.com' do
   roles 'frontend'
   addons %w(opscode-reporting opscode-push-jobs-server opscode-manage)
+  machine_options vagrant_options: {
+    'vm.box' => 'opscode-centos-6.6',     
+    'vm.network' => ":private_network, :ip => '33.33.33.20'"
```
Comment (ncerny) [inline recipes/default.rb:33]: NIT: don't use "" if you don't need interpolation.

File: recipes/default.rb [hunk 3]
```diff
@@ -11,6 +11,10 @@
 chef_server 'be01.vagrantup.com' do
   roles 'backend'
   addons %w(opscode-reporting opscode-push-jobs-server)
+  machine_options vagrant_options: {
+      'vm.box' => 'opscode-centos-6.6',     
+      'vm.network' => ":private_network, :ip => '33.33.33.10'"
```
Comment (ncerny) [inline recipes/default.rb:16]: NIT: don't use "" if you don't need interpolation.

**Pull request merged**